packages/@okta/vuepress-site/docs/reference/api/idps/index.md

```diff
@@ -5124,7 +5132,8 @@ Protocol settings for authentication using the [OpenID Connect Protocol](http://
 
 | Property    | Description                                                      | DataType                                          | Nullable | Readonly | MinLength |
 | ----------- | ---------------------------------------------------------------- | ------------------------------------------------- | -------- | -------- | --------- |
-| credentials | Client authentication credentials for an [OAuth 2.0 Authorization Server (AS)](https://tools.ietf.org/html/rfc6749#section-2.3) | [Credentials object](#oauth-2-0-and-openid-connect-credentials-object) | FALSE | FALSE |   |
+|algorithms   | Settings for signing authorization requests                      | [OIDC Algorithms object](#oidc-algorithms-object) | TRUE    | FALSE    |
+|credentials  | Client authentication credentials for an [OAuth 2.0 Authorization Server (AS)](https://tools.ietf.org/html/rfc6749#section-2.3) | [Credentials object](#oauth-2-0-and-openid-connect-credentials-object) | FALSE | FALSE |   |
```

Comment:
don't think we need to introduce the abbreviation for authorization server here
```suggestion
|credentials  | Client authentication credentials for an [OAuth 2.0 Authorization Server](https://tools.ietf.org/html/rfc6749#section-2.3) | [Credentials object](#oauth-2-0-and-openid-connect-credentials-object) | FALSE | FALSE |   |
```

Reply:
Hmm, looking at this again, I think I only added the row `algorithms` on top of the row `credentials`, I did not change or update the `credentials` row. In this case, would you still want me to remove the `(AS)`?

Reply:
https://developer.okta.com/docs/reference/api/idps/#oauth-2-0-protocol

Reply:
if you don't mind and it doesn't cause any problems....it's not necessary